.gitlab/common/test_infra_version.yml

```diff
@@ -5,5 +5,5 @@
 
 ---
 variables:
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 60f7f2c30806
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 0686f6b8dd0c
```

Comment:
https://github.com/DataDog/datadog-agent/pull/34796
looks like `test-infra` has been bumped again, up to you if you want to bump the version to latest / just merge this in and the auto-version-bump will update it later!